cfpb/cfgov-refresh
# Migrate Spanish Ask to English templates

Moves Spanish Ask content into shared templates with English Ask. 

## Additions

- Translations in `django.po` for hard-coded content in templates that will now be used on Spanish pages
- Script to migrate formatted first paragraphs into short answer field on Spanish answer pages
- Spanish mega menu vars file
- Spanish logo images


## Removals

- Spanish Ask templates
- Spanish answer page print view
- Virtual pageview event for Ask answer pages
- Customization of Ask breadcrumbs based on referrer


## Changes

- Update English Ask templates to also work for Spanish content
- Update Ask page models to use a single shared template for English and Spanish
- Update header / footer templates to create a Spanish version that will be applied to all pages whose language is set to Spanish in Wagtail

## Testing

1. Check out branch & run `python cfgov/manage.py runscript migrate_spanish_short_answers`
2. Navigate to [Spanish Ask landing page](http://localhost:8000/es/obtener-respuestas/), [Spanish Ask category page](http://localhost:8000/es/obtener-respuestas/categoria-comprar-un-vehiculo/), and [Spanish Ask answer page](http://localhost:8000/es/obtener-respuestas/que-efecto-tendria-sobre-mi-credito-comparar-diferentes-ofertas-de-prestamos-para-vehiculos-es-763/) and 
    - verify that they display/function like their English counterparts but with a Spanish header & footer
    - verify that first paragraph on answer page displays as larger text
3. Navigate to a [non-Ask Spanish page](http://localhost:8000/es/prepararse-para-comprar-casa/) and verify that it also displays the Spanish header/footer

## Screenshots

### Spanish answer page
![spanish-answer-page](https://user-images.githubusercontent.com/778171/56327249-70c72500-612e-11e9-84de-63015ecc109d.png)
